HOWTO_RELEASE.md

```diff
@@ -0,0 +1,14 @@
+Release procedure is as follows:
+
+- Let $vmaj, $vmin and $vpat be Major, Minor and Patch version numbers
+- Drop "dev" suffix from $vpat
+- Let $ver be "$vmaj.$vmin.$vpat"
+- Make sure Version variable is set correctly in main.go ($ver)
+- Compile CHANGELOG.md section for $ver
+  (please only include user-relevant changes, and be concise)
+- `git tag -a v$ver`
+- If this is a .0 release:
+  - `git branch release/v$vmaj.$vmin`
+  - Update Version in new branch ( vpat++ )
+- Update Version in master branch ( vmin++; vpat=0-dev )
```

Comment:
Please add a line below this that Drone automatically creates the Release and that it shouldn't be created manually (it doesn't break if one does, but just to document that it's not necessary)

Reply:
can you push directly to the branch associated to this PR ?